src/css/popup.css

```diff
@@ -479,17 +494,21 @@ onboarding-panel {
 .email-domain-illustration {
   margin: 0 auto;
   display: block;
+  height: 80px;
```

Comment:
Sets a fixed height for the image to make sure all of the UI components fit (including the new email trackers removed data), since there isn't a lot of real estate on the popup panel.